src/navigation/sim_planning/sim_planning/node_map.py

```diff
@@ -246,23 +246,23 @@ def main(args=sys.argv[1:]):
     print_logs = False
     spline_len = 3999
 
-    # # processing args
-    # opts, arg = getopt.getopt(args, str(), ['log=', 'print_logs', 'length='])
-
-    # # TODO: provide documentation for different options
-    # for opt, arg in opts:
-    #     if opt == '--log':
-    #         loglevel = arg
-    #     elif opt == '--print_logs':
-    #         print_logs = True
-    #     elif opt == '--length':
-    #         spline_len = arg
-    # # validating args
+    # processing args
+    opts, arg = getopt.getopt(args, str(), ['log=', 'print_logs', 'length=', 'ros-args'])
```

Comment:
it's fine, but I'd prefer the use of ros args here (even though it makes the commands a bit more clunky). It just makes the code a bit more standard (since its using the ros parameter interface) and also means you can control the args easily in launch files.

Reply:
Something to think about for the next node that gets built

Reply:
yeah even this can be ammended when we have launch files set for everything. currently lidar also uses these logs but ideally its the ros parameters